platform/m480/src/debug_stdiouart.c

```diff
@@ -0,0 +1,438 @@
+/*
+ * Copyright (c) 2016-2017, Nuvoton, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <uvisor.h>
+
+/* Abstract from M480.h */
```

Comment:
Hi @ccli8,

We are working on making debugging without semihosting easier. We unfortunately can't accept the UART driver for M480 into uVisor, but you are free to continue overriding `default_putc` out-of-tree.

Thanks